src/stores/CustomRoomTagStore.js

```diff
@@ -48,7 +48,7 @@ class CustomRoomTagStore extends EventEmitter {
     constructor() {
         super();
         // Initialise state
-        this._state = {tags: this._getUpdatedTags()};
+        this._state = {tags: {}};
```

Comment:
not sure I get why this isn't necessary - do we know the store will always spit out an update?

Reply:
because it updates on sync. Also it's unlikely that anything useful is available when the bundle has just loaded as this is a singleton that gets created when the module is loaded (that's why I removed it).

Reply:
ah, makes sense then